src/c++/perf_analyzer/infer_context.cc

```diff
@@ -53,7 +53,7 @@ InferContext::SendInferRequest(bool delayed)
   if (using_json_data_) {
     UpdateJsonData();
   }
-  SendRequest(request_id_++, delayed);
+  SendRequest(request_id_++, delayed, /* ignored */ 0);
```

Comment:
Can we have two function prototypes where one calls the other? Or a default value? I don't like the caller needing to know that the value is ignored and to put in a value

Reply:
And, IS it actually ignored? It looks like it is still put into the structs, as it should be.
Note that it is true that seq_id of 0 means "not a sequence". This is true in triton, not just our code.

Reply:
Added default value. And good point. Ignored was a bit of a stretch to say here. What I meant is that it will not eventually be written out to the profile export file for non-sequence models.